lib/members.rb

```diff
@@ -0,0 +1,28 @@
+require 'nokogiri'
+require 'open-uri/cached'
+require 'field_serializer'
+require_relative 'member'
+OpenURI::Cache.cache_path = '.cache'
+
+class Members
+  include FieldSerializer
+
+  def initialize(noko)
+    @noko = noko
+  end
+
+  field :members do
+    legislature_members
+  end
+
+  def legislature_members
+    noko.xpath('//div[@class="entry"]/table[1]/tbody/tr').map do |row|
+      Member.new(row).to_h
+    end
+  end
```

Comment:
It's not obvious why you have a field here that is simply calling another public method.